src/backend/distributed/commands/function.c

```diff
@@ -65,6 +65,13 @@
 #include "utils/syscache.h"
 #include "utils/regproc.h"
 
+/*
```

Comment:
Can you remind me which NOTICE? `LookupFuncWithArgs` doesn't seem to produce any notices. It is mostly erroring out.

Also, would be nice to remind which test was failing and its output?

Reply:
call stack is shown below:

```
LookupFuncWithArgs
   => LookupTypeNameOid
       => LookupTypeName
           => LookupTypeNameExtended
```

Notice log is
```
/* emit nuisance notice (intentionally not errposition'd) */
ereport(NOTICE,
	(errmsg("type reference %s converted to %s",
			TypeNameToString(typeName),
			format_type_be(typoid))));
```

Reply:
Can we instead do
```
OldValue = ClientMinMessage;

-- suppress NOTICE if running under vanialla tests
SetLocalClientMinMessagesIfEnableUnsupportedFeatureMessages(Warning);
-- force type resolve
FunctionToObjectAddress()

-- set it back  
SetLocalClientMinMessagesIfEnableUnsupportedFeatureMessages(OldValue);
```

Which approach you think better? I'd rather avoid adding a new GUC and logic if possible.

Reply:
This was an approach I was looking for.